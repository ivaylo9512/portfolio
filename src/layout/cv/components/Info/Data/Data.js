import TabLink from '@cv/components/TabLink/TabLink';
import User from '@svgIcons/User';
import Name from '@svgIcons/Name';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faLocationDot, faCalendarDays, faPhone } from "@fortawesome/free-solid-svg-icons"
import { faGithub } from "@fortawesome/free-brands-svg-icons"

const Data = () => {
    return(
        <div className='data'>
            <div className='name'>
                <User />
                <Name />
            </div>
            <div className="info">
                <div className='locationn'>
                    <div className='icon'>
                        <FontAwesomeIcon icon={faLocationDot} />
                    </div>
                    Sofia
                </div>
                <div>
                    <div className='icon'>
                    <FontAwesomeIcon icon={faGithub} />
                    </div>
                    <TabLink shouldMatch={true} href={'https://github.com/ivaylo9512'}>
                        Github
                    </TabLink>
                </div>
                <div>
                    <div className='icon'>
                        <FontAwesomeIcon icon={faCalendarDays} />
                    </div>
                    27
                </div>
                <div className="phone">
                    <div className='icon'>
                        <FontAwesomeIcon icon={faPhone} />
                    </div>
                    <span>+359882603703</span>
                </div>
            </div>
        </div>     
    )
}

export default Data