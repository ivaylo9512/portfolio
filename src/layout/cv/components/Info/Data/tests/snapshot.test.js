import { shallow } from 'enzyme';
import Data from '../Data';

describe('Data snapshot tests', () => {
    const createWrapper = () => shallow(
        <Data />
    );

    it('should match snapshot', () => {
        const wrapper = createWrapper();

        expect(wrapper).toMatchSnapshot();
    })
})