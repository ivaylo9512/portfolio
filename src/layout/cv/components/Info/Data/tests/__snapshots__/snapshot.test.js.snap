// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data snapshot tests should match snapshot 1`] = `
<div
  className="data"
>
  <div
    className="name"
  >
    <User />
    <InfoName />
  </div>
  <div>
    <Brush />
    Sofia
  </div>
  <div>
    <TabLink
      href="https://github.com/ivaylo9512"
      shouldMatch={true}
    >
      Github
    </TabLink>
    <ProfileGit />
  </div>
  <div>
    <Age />
    26
  </div>
  <div
    className="phone"
  >
    0886892842
    <Phone />
  </div>
</div>
`;
